jaeger-crossdock/src/test/java/com/uber/jaeger/crossdock/resources/behavior/http/EndToEndBehaviorResourceTest.java

```diff
@@ -21,18 +21,17 @@
  */
 package com.uber.jaeger.crossdock.resources.behavior.http;
 
+import static org.junit.Assert.*;
+
 import com.uber.jaeger.Span;
 import com.uber.jaeger.crossdock.api.*;
 import com.uber.jaeger.reporters.InMemoryReporter;
 import com.uber.jaeger.samplers.ConstSampler;
 import io.opentracing.Tracer;
+import java.util.*;
```

Comment:
I'm not convinced that the `verification` step fails builds for style failures because it passed this build.